docs/03-tutorials/00-serverless/svls-01-create-inline-function.md

```diff
@@ -129,9 +129,9 @@ Follow these steps:
 
 2.  In the left navigation panel, go to **Workloads** > **Functions** and select **Create Function**.
 
-3.  In the pop-up box, provide the Function's name or leave the autogenerated one. Select the `Node.js 12` runtime, and press **Create** to confirm changes.
+3.  In the dialog box, provide the Function's name or leave the autogenerated one. Select the `Node.js 12` runtime, and press **Create** to confirm changes.
```

Comment:
```suggestion
3.  In the dialog box, provide the Function's name or leave the autogenerated one. Select the `Node.js 14` runtime, and press **Create** to confirm changes.
```

or leave the default Runtime?

Reply:
@kwiatekus what do you say?

Reply:
Alright, let's discuss the default runtime another day. Committing Wojciech's suggestion.